server/views/pages/activities/record-attendance/attendance-list-single.njk

```diff
@@ -12,14 +12,19 @@
 {% from "partials/attendance/otherEvent.njk" import otherEvent %}
 {% from "partials/statusBasedCellLocation.njk" import statusBasedCellLocation %}
 {% from "partials/showProfileLink.njk" import showProfileLink %}
+{% from "partials/formatSessions.njk" import formatSessions %}
 {% from "partials/service-user-name.njk" import serviceUserName %}
 
 {% set pageTitle = applicationName + " - Record attendance" %}
 {% set pageId = 'attendance-list-page' %}
-{% set jsBackLink = true %}
 {% set cancelledComment = '' %}
 {% if instance.comment !== undefined and instance.comment !== null %}
- {% set cancelledComment = " - "+instance.comment %}
+  {% set cancelledComment = " - "+instance.comment %}
+{% endif %}
+{% set activityDate = instance.date | toDate | formatDate %}
+{% if selectedSessions.length > 0 %}
+    {% set hardBackLinkText = 'Go back to activities for ' + activityDate + ' - ' + formatSessions(selectedSessions) %}
+    {% set hardBackLinkHref = '/activities/attendance/activities?date=' + session.recordAttendanceRequests.activityDate + '&sessionFilters=' + selectedSessions | join(",") %}
```

Comment:
Builds the text and url for the anchor to sh that it can return to the original activities view.